OAuth/ResourceOwner/AbstractResourceOwner.php

```diff
@@ -301,9 +301,9 @@ protected function generateNonce()
     /**
      * Configure the option resolver
      *
-     * @param OptionsResolverInterface $resolver
+     * @param OptionsResolver $resolver
      */
-    protected function configureOptions(OptionsResolverInterface $resolver)
+    protected function configureOptions(OptionsResolver $resolver)
```

Comment:
@stloyd Finally seems to not brake BC.

Looks at the new `SpotifyResourceOwner` on my branch: https://github.com/NewsConnect/HWIOAuthBundle/blob/deprecated-fixes/OAuth/ResourceOwner/SpotifyResourceOwner.php#L54

This use old signature but still passing the tests.